src/Components/ReviewQuestion/ReviewQuestion.tsx

```diff
@@ -0,0 +1,93 @@
+import styled from 'styled-components';
+import { CustomRadio } from '../CustomRadio/CustomRadio';
+
+interface ReviewQuestionProps {
+  title: string;
+  contents: string[];
```

Comment:
```suggestion
  contents: [string, string];
```

현재 단 두 개의 항목만 사용하니 튜플로 쓰는 게 더 명확할 것 같아요!

Reply:
반영했습니다!!